tools/report-converter/codechecker_report_converter/analyzer_result.py

```diff
@@ -72,6 +72,14 @@ def _add_report_hash(self, plist_objs):
 
     def _get_analyzer_result_file_content(self, result_file):
         """ Return the content of the given file. """
+        if not os.path.exists(result_file):
+            LOG.error("Result file does not exists: %s", result_file)
+            return
+
+        if os.path.isdir(result_file):
+            LOG.error("Directory is given instead of a file: %s", result_file)
+            return
```

Comment:
What about the other analyzer tools which emit a directory?

Reply:
As the documentation of this function says this function will "Return the content of the given file". So it expects a file not a directory.

Reply:
Ok. I thought that infer would use this function too, but it doesn't.